strings/python/palindrome.py

```diff
@@ -22,9 +34,6 @@ def is_palindrome(s: str):
 
```

Comment:
Doc test Statements look fine. Returning Everything as expected. I think this can be Merged.